packages/playground/src/components/pull-request/pull-request-comment-box.tsx

```diff
@@ -22,7 +22,7 @@ const PullRequestCommentBox: React.FC<PullRequestCommentBoxProps> = ({ onSaveCom
   return (
     <div className="flex gap-x-2">
       <Avatar size="6">
-        {/* <AvatarImage src={user?.avatarUrl} /> */}
+        <AvatarImage src={AvatarUrl} />
```

Comment:
We're adding a placeholder hardcoded image as avatar?

Reply:
Yeah just as a placeholder